src/annotations_query_ctrl.ts

```diff
@@ -7,7 +7,7 @@ export class KustoDBAnnotationsQueryCtrl {
 
   defaultQuery = '<your table>\n| where $__timeFilter() \n| project TimeGenerated, Text=YourTitleColumn, Tags="tag1,tag2"';
 
-  /** @ngInject **/
+  /** @ngInject */
```

Comment:
I think it's:  /* @ngInject */

Reply:
Are you sure? https://github.com/search?q=org%3Agrafana+%2F**+ngInject+*%2F&type=Code

Reply:
I guess it doesn't matter.  I've always used it like the ng-annotate docs suggest:  https://www.npmjs.com/package/babel-plugin-angularjs-annotate?activeTab=readme